test/unit/contextRequests/blackButtonCloseRequest.json

```diff
@@ -11,6 +11,11 @@
           "type": "string"
         },
         {
+          "name": "aux_op_status",
+          "value": "CLOSED",
```

Comment:
In fact, it is already documented as an issue at https://github.com/telefonicaid/iotagent-thinking-things/issues/50 :)